dm/syncer/syncer.go

```diff
@@ -2285,7 +2285,7 @@ func (s *Syncer) handleQueryEvent(ev *replication.QueryEvent, ec eventContext, o
 	// TODO: save stmt, tableName to avoid parse the sql to get them again
 	qec.p = parser.New()
 	for _, sql := range qec.splitDDLs {
-		sqls, err2 := s.processOneDDL(qec, sql)
+		sqls, err2 := s.processOneDDL(qec, sql, ev.StatusVars)
```

Comment:
maybe add `ev.StatusVars` to `qec` is better?

Reply:
yes, it is more appropriate.